shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -49,20 +57,25 @@ public PlatformViewsController() {
         vdControllers = new HashMap<>();
     }
 
-    public void attachFlutterView(FlutterView view) {
-        if (mFlutterView != null)
+    public void attach(Context context, TextureRegistry textureRegistry, BinaryMessenger messenger) {
```

Comment:
nit: Javadoc on this method and `detach` would be helpful. The params could use the explanations that you've given them where you've declared them as private members.